website/docs/reference/data-test-configs.md

```diff
@@ -296,3 +300,40 @@ models:
 ```
 
 Given the config, the data test runs on a different Snowflake virtual warehouse than the one in your default connection to enable better price-performance with a different warehouse size or more granular cost allocation and visibility.
+
+#### Add a description to generic and singular tests
+
+Starting from dbt v1.9 (and also versionless in dbt Cloud), you can add [descriptions](/reference/resource-properties/data-tests#description) to both generic and singular tests.
```

Comment:
We changed the naming here. Might be beneficial to link out to the new docs too. 
```suggestion
Starting from dbt v1.9 (and dbt Cloud [release tracks](/docs/dbt-versions/cloud-release-tracks)), you can add [descriptions](/reference/resource-properties/data-tests#description) to both generic and singular tests.
```